student.rb

```diff
@@ -1,13 +1,12 @@
-require_relative './person.rb'
+require_relative 'person'
 
 class Student < Person
-  def initialize(id, age, classroom, name = "Uknown", parent_permission = true)
-    super(id, age, name, parent_permission)
+  def initialize(classroom)
+    super(age, name, parent_permission: true)
     @classroom = classroom
   end
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: 

- [ ] When you inherit from a parent class and want to call the parent class's constructor using `super`, it's a good practice to accept the arguments in the `initialize method` of the child class and then pass them down to the parent class's constructor. This ensures that the child class properly initializes its own attributes and properly passes relevant data to the parent class. So I'll suggest you accept all the arguments in the initialize method of the `Student class` Before passing it down to the super. :+1: :+1: You should update your code to reflect this. :+1: 

```sh
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```